katportalclient/client.py

```diff
@@ -761,26 +786,43 @@ def future_targets_detail(self, id_code):
         raise tornado.gen.Return(targets_list)
 
     def set_reference_observer_config(
-            self, longitude, latitude, altitude, timestamp):
+            self, longitude=None, latitude=None, altitude=None, timestamp=None):
         """
         Set the reference observer config to be used when retrieving
         the future target list from a schedule block.
 
+        .. note::
+
+            If longitude, latitude or altitude is None then katpoint will use
+            the array's default reference observer. If timestamp is None,
+            katpoint will use the current utc time to calculate the pointing
+            details.
+
         Parameters
         ----------
         longitude: float
             The longitude of the reference observer used in calculating the
             pointing details.
```

Comment:
Good point. I added this in another branch that was branched from this one - https://github.com/ska-sa/katportalclient/commit/9ac1d309b2c07f96d1b552985fc5a467b45a04f1